### PR TITLE
Tried to fix hud colors

### PR DESCRIPTION
**Pull request by T1MOXA**
Original idea - https://github.com/ValveSoftware/csgo-osx-linux/issues/2233#issuecomment-555558114

### CODE REVIEW

**Comment by sneak-it** [comment]
I would rather no colors than an awful translation string pinned to the top of the HUD.

File: addons/sourcemod/scripting/shavit-hud.sp [hunk 1]
```diff
@@ -1401,14 +1401,14 @@ void UpdateMainHUD(int client)
...
-		StrCat(sBuffer, 512, "<pre>");
+		StrCat(sBuffer, 512, "#SFUI_Notice_NewWaveBegun_\n<pre>");
 		iLines = AddHUDToBuffer_CSGO(client, huddata, sBuffer, 512);
 		StrCat(sBuffer, 512, "</pre>");
 	}
 
 	if(iLines > 0)
 	{
-		PrintHintText(client, "%s", sBuffer);
+		PrintCenterText(client, "%s", sBuffer);
```
Comment (Bara) [inline addons/sourcemod/scripting/shavit-hud.sp]: I would add `#SFUI_Notice_NewWaveBegun_\n` here.

**Comment by T1MOXA** [comment]
image

**Comment by shavitush** [comment]
Not acceptable unfortunately. As @sneak-it said, rather not have colors at all than this

**Comment by shavitush** [comment]
Also, your commit breaks the HUD for CS:S.

**Comment by kidfearless** [comment]
Since it's been a few months since the last comment I'm going to close this. At this point, most people have downloaded the [hud fix plugin](https://github.com/Franc1sco/FixHintColorMessages) made by franc1sco.

**Pull request closed without merging**